fakesqs/metadata.rb

```diff
@@ -0,0 +1,6 @@
+name 'fakesqs'
+
+supports 'ubuntu'
+
+depends 'ruby-brightbox'
```

Comment:
@gilleyj My only question would be what you need this ruby for? Is the one in our vagrant boxes not enough?